### PR TITLE
user BasicEdit Vuefication

### PR DESCRIPTION
**Pull request by sarvesh211999**
Vuefication of user.basicedit.tpl
related to #2009 issue

### CODE REVIEW

**Comment by sarvesh211999** [comment]
@lhchavez @SpaceWhite Review

File: frontend/www/js/omegaup/components/user/BasicEdit.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      </form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {OmegaUp, T, API} from '../../omegaup.js';
+export default {
+  data: function() {
+    return { T: T, newPassword1: "", newPassword2: "", username: "", }
```
Comment (lhchavez) [inline frontend/www/js/omegaup/components/user/BasicEdit.vue]: nit: can you use single quotes consistently in JavaScript?

File: frontend/www/js/omegaup/components/user/BasicEdit.vue [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    var self = this;
+    omegaup.API.User.profile()
+        .then(function(data) { self.username = data.userinfo.username; })
+        .fail(omegaup.UI.apiError);
+  },
+  methods: {
+    formSubmit: function() {
+      var self = this;
+      if (self.newPassword1 != self.newPassword2) {
+        omegaup.UI.error('Los passwords nuevos deben ser iguales.');
```
Comment (lhchavez) [inline frontend/www/js/omegaup/components/user/BasicEdit.vue]: do you mind adding a translation string?

File: frontend/templates/user.basicedit.tpl [hunk 3]
```diff
@@ -1,40 +1,6 @@
...
-						<button type="submit" class="btn btn-primary">{#wordsSaveChanges#}</button>
-					</div>
-				</div>
-			</form>
-		</div>
-	</div>
-
+	<div id="user-basic-edit"></div>
+	<script type="text/javascript" src="{version_hash src="/js/dist/user_basic_edit.js"}"></script>
 	<script type="text/javascript" src="{version_hash src="/js/user.basicedit.js"}"></script>
```
Comment (SpaceWhite) [inline frontend/templates/user.basicedit.tpl]: I think we don't need this anymore

File: frontend/www/js/omegaup/user/basicedit.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import Vue from 'vue';
+import user_BasicEdit from '../components/user/BasicEdit.vue';
```
Comment (SpaceWhite) [inline frontend/www/js/omegaup/user/basicedit.js]: nit: can you change this to `UserBasicEdit`?

File: frontend/www/js/omegaup/components/user/BasicEdit.vue (hunk 2 above)
Comment (sarvesh211999) [reply frontend/www/js/omegaup/components/user/BasicEdit.vue]: Yeah Sure

File: frontend/templates/user.basicedit.tpl (hunk 3 above)
Comment (sarvesh211999) [reply frontend/templates/user.basicedit.tpl]: Yeah

File: frontend/www/js/omegaup/user/basicedit.js (hunk 4 above)
Comment (lhchavez) [reply frontend/www/js/omegaup/user/basicedit.js]: `user_BasicEdit` was correct :(

**Comment by SpaceWhite** [comment]
Thanks for your contribution :) @sarvesh211999

**Pull request merged**